index.js

```diff
@@ -1,12 +1,28 @@
 'use strict';
 const uniqueRandomArray = require('unique-random-array');
 const pokemon = require('./pokemon');
+const pokemonGerman = require('./pokemon_de');
 
 exports.all = pokemon;
 exports.random = uniqueRandomArray(pokemon);
 
-exports.getName = id => {
-	const name = pokemon[id - 1];
+function getLocalizedPokemonList(lang) {
+	let list;
+
```

Comment:
Put the `require` call inline for the localization. Don't want the overhead of loading every localization each time.

Can also simplify the code by just prepending the `lang` to the filename and then `require` it.